app/services/imports/sales_logs_import_service.rb

```diff
@@ -432,9 +432,11 @@ def other_people_with_details(xml_doc, attributes)
     end
 
     def household_members(_xml_doc, attributes)
-      return attributes["hholdcount"] + 2 if attributes["jointpur"] == 1
-
-      attributes["hholdcount"] + 1 if attributes["jointpur"] == 2
+      if attributes["jointpur"] == 2
```

Comment:
Existing tests didn't break 🫠
It's a good point tho, I couldn't think of how to test it, but I've just added a test. 
It basically hangs because it tries to complete other calculations involving hhmemb without it being set. So that test hangs with the old version, but runs fine with the changes in this PR